lib/python_prefida/get_version.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import os
+
+
+def get_version(fidasim_dir):
```

Comment:
I remember you asked about getting the git hash in python. Something like this will work
```python
import subprocess
subprocess.check_output(['git','--git-dir=/u/lstagner/FIDASIM/.git','describe','--tags','--always','--dirty'])
```
just make sure to check that git is installed and the `.git` directory exists first.